source/user-manual/capabilities/command-monitoring/how-it-works.rst

```diff
@@ -34,4 +34,4 @@ Example::
 Process the output
 ------------------
 
-After configuring the system to monitor the command's output as if it were log data, custom rules can be created, like for :ref:`Log analysis <manual_log_analysis>` for instance, in order to process the output and trigger an alert when alert criteria are met.
+After configuring the system to monitor the command's output as if it were log data, custom rules can be created, like for :ref:`Log analysis <manual_log_analysis>` for instance, in order to process the output and trigger an alert when alert criteria are met. Read the `Scheduling remote commands for Wazuh agents <https://wazuh.com/blog/scheduling-remote-commands-for-wazuh-agents//>`_ document for more information and remote command use cases.
```

Comment:
I wonder if we could emphasize these links through a `.. note::` section.

Reply:
Links to blog are now under a `.. note::` section.